cql3/selection/selector_factories.hh

```diff
@@ -105,6 +106,22 @@ public:
         return _factories[0]->is_count_selector_factory();
     }
 
+    bool does_reduction() const {
+        if (_factories.size() != 1) {
+            return false;
```

Comment:
Why? Does SELECT min(x), max(x) not work?

Reply:
I was developing the count(*) parallelization which has this limitation and I wanted to focus on distributing UDAs/native aggregates. Probably it isn't hard to add support to multiple selection, however I wasn't investigating much into this.

Reply:
If we change it later, it will require a schema feature. So better to extend it now, if it's easy.

Reply:
Extended